CMakeLists.txt

```diff
@@ -580,22 +580,6 @@ include(iree_plugin_register)
 set(IREE_PACKAGE_ROOT_DIR "${CMAKE_CURRENT_LIST_DIR}")
 set(IREE_PACKAGE_ROOT_PREFIX "iree")
 
-#-------------------------------------------------------------------------------
-# Experimental ROCM HAL driver
-# Enable with: -DIREE_EXTERNAL_HAL_DRIVERS=rocm
```

Comment:
Do we want to wait to merge this until issues like https://github.com/openxla/iree/issues/17033 are resolved? Maybe set a timeline on deprecation/removal of 1-2 weeks? We already have a few PRs in flight that touch this experimental code, so would be good to cut over ASAP and not need to maintain code scheduled for removal.

Reply:
Let's wait till next week to see if any more issues are reported. I'll take a look at the reported one and issue a PSA to use only HIP from now so we can stress test it further.